test/runtests.jl

```diff
@@ -82,27 +76,27 @@ println("Read test.feather into julia...")
 run(`docker cp feathertest:/home/test.feather test.feather`)
 df = Feather.read("test.feather")
 
-@test isequal(df[:Autf8], NullableArray(["hey","there","sailor"]))
-@test isequal(df[:Abool], NullableArray([true, true, false]))
-@test isequal(df[:Acat], CategoricalArrays.NullableCategoricalArray(["a","b","c"]))
-@test isequal(df[:Acatordered], CategoricalArrays.NullableCategoricalArray(["d","e","f"]))
-@test isequal(df[:Adatetime], NullableArray([DateTime(2016,1,1), DateTime(2016,1,2), DateTime(2016,1,3)]))
-@test isequal(df[:Afloat32], NullableArray(Float32[1.0, 0.0, 0.0], [false, true, false]))
-@test isequal(df[:Afloat64], NullableArray([Inf,1.0,0.0]))
+@test df[:Autf8] == ["hey","there","sailor"]
+@test df[:Abool] == [true, true, false]
+@test df[:Acat] == CategoricalArray(["a","b","c"])
```

Comment:
`== ["a","b","c"]` would have the same effect. Maybe you want to test the type too? Same in the other block below.